### ISSUE TITLE
chat disappears too fast

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#1408]
Team chat (the most important text), normal chat (arguably important) and all system messages are mixed into a single list which really go fast, with too few lines.

I think it would be good to divide that text in several areas, so that important messages are not lost that fast.

**Comment by Calinou** [comment]
If chat only stays on for 1-2 seconds, this is likely due to some kind of error spam which means old lines are flushed very quickly.

Still, I believe the default chat duration is quite short (Quake 3 defaulted `con_notifytime` to 3). I'd probably increase it to 15 seconds at least.

**Comment by bmorel** [comment]
even with longer time, they will still be scrolled out by less important messages, so I think even with greater default time, it would still disappear too fast when there's lot of action, and thus, still be a problem.

**Comment by Gireen** [comment]
illwieckz mentioned somewhere that it would be good to have a separate chat log that could be displayed in the scoreboard.

It might in general be nice to have a way to display filtered console informations, like a killfeed, team messages or game events.

Then we could for example also display a chatlog with recent messages above the chat input.

**Comment by illwieckz** [comment]
Yes, Quake Live displays the latest chat lines in the scoreboard, and Source-based games display the latest chat messages above the the chat input. Having both would be great. See #830.
